miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/MisoWebUtils.java

```diff
@@ -59,28 +59,22 @@
 public class MisoWebUtils {
   private static final Logger log = LoggerFactory.getLogger(MisoWebUtils.class);
 
-  public static String generateErrorDivMessage(String errorMessage) {
-    return "<div id='errordiv' class='flasherror'>" + errorMessage + "</div>";
-  }
-
-  public static String generateErrorDivMessage(String errorMessage, String exceptionMessage) {
-    return "<div id='errordiv' class='flasherror'>" + errorMessage + "<br/><pre>" + exceptionMessage + "</pre></div>";
-  }
-
-  public static Map<String, String> checkStorageDirectories(String baseStoragePath) {
+  public static Map<String, String> checkStorageDirectories(String baseStoragePath, String fileStoragePath) {
     Map<String, String> checks = new HashMap<>();
     if (baseStoragePath.endsWith("/")) {
       try {
         File misoDir = new File(baseStoragePath);
         if (LimsUtils.checkDirectory(misoDir, true)) {
-          LimsUtils.checkDirectory(new File(baseStoragePath, "files"), true);
-          LimsUtils.checkDirectory(new File(baseStoragePath, "files/submission"), true);
           LimsUtils.checkDirectory(new File(baseStoragePath, "log"), true);
           LimsUtils.checkDirectory(new File(baseStoragePath, "temp"), true);
-          checks.put("ok", "All storage directories OK");
+          if (LimsUtils.checkDirectory(new File(fileStoragePath), true)) {
+            LimsUtils.checkDirectory(new File(fileStoragePath, "submission"), true);
+            checks.put("ok", "All storage directories OK");
+          } else {
+            checks.put("error", "Error accessing MISO storage files directory.");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "error" 5 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)